src/Rel_PT_Emission_Profile.jl

```diff
@@ -59,15 +68,29 @@ R_isco = r_isco(a_star, M)
 L_edd = 3e4*L_☼*(M/M_☼)
 Mdot = -L_edd/(c^2*η)
 mdot = 0.1*Mdot
+g = 500
```

Comment:
I'd have to double check, but isn't `g` the redshift factor in Fanton et al., roughly in the range 0 to 2 -- where did you get the value of 500 from?

Reply:
You're right about the g factor, but we didn't want to go into the detail of calculating it quite yet, so we were just messing around with placeholder values so far. The 500 was just the last one we tried to see what it would look like, there's no real signifance to it.